README

```diff
@@ -44,3 +44,14 @@
       -1 Don't re-run command if files changed while command was running
       -s Run command immediately at start
 
+    ~ Environment variables
+
+      when-changed provides the following environment variables:
+
+      - WHEN_CHANGED_EVENT: reflects the current event type that occures.
```

Comment:
Can you mention WHEN_CHANGED_FILE as well?

Reply:
Done